pkg/terraform/v1beta2/update.go

```diff
@@ -74,21 +74,27 @@ func updateAzureWorkerset(existingWorkerSet *kubeonev1beta2.DynamicWorkerConfig,
 	}
 
 	flags := []cloudProviderFlags{
-		{key: "assignPublicIP", value: azureCloudConfig.AssignPublicIP},
-		{key: "availabilitySet", value: azureCloudConfig.AvailabilitySet},
 		{key: "location", value: azureCloudConfig.Location},
 		{key: "resourceGroup", value: azureCloudConfig.ResourceGroup},
+		{key: "vnetResourceGroup", value: azureCloudConfig.VNetResourceGroup},
+		{key: "vmSize", value: azureCloudConfig.VMSize},
+		{key: "vnetName", value: azureCloudConfig.VNetName},
+		{key: "subnetName", value: azureCloudConfig.SubnetName},
+		{key: "loadBalancerSku", value: azureCloudConfig.LoadBalancerSku},
 		{key: "routeTableName", value: azureCloudConfig.RouteTableName},
+		{key: "availabilitySet", value: azureCloudConfig.AvailabilitySet},
+		{key: "assignAvailabilitySet", value: azureCloudConfig.AssignAvailabilitySet},
 		{key: "securityGroupName", value: azureCloudConfig.SecurityGroupName},
 		{key: "zones", value: azureCloudConfig.Zones},
 		{key: "imagePlan", value: azureCloudConfig.ImagePlan},
-		{key: "subnetName", value: azureCloudConfig.SubnetName},
-		{key: "tags", value: azureCloudConfig.Tags},
-		{key: "vmSize", value: azureCloudConfig.VMSize},
-		{key: "vnetName", value: azureCloudConfig.VNetName},
+		{key: "imageReference", value: azureCloudConfig.ImageReference},
 		{key: "imageID", value: azureCloudConfig.ImageID},
 		{key: "osDiskSize", value: azureCloudConfig.OSDiskSize},
+		{key: "osDiskSKU", value: azureCloudConfig.OSDiskSKU},
 		{key: "dataDiskSize", value: azureCloudConfig.DataDiskSize},
+		{key: "dataDiskSKU", value: azureCloudConfig.DataDiskSKU},
+		{key: "assignPublicIP", value: azureCloudConfig.AssignPublicIP},
+		{key: "tags", value: azureCloudConfig.Tags},
```

Comment:
please sort it

Reply:
I've sorted them in the same way as in [machine-controller](https://github.com/kubermatic/machine-controller/blob/7fcde862813596277b8a819ea0591f3b86d8722b/pkg/cloudprovider/provider/azure/types/types.go#L24-L53) because otherwise comparing structs is difficult and error-prone.